### DISCUSSION TITLE
is there any method to use marp on sublime-text?

### DISCUSSION THREAD

**Discussion by dreistein543** [marp-team/marp#381] [Ideas] [answered]
I have tried marp on `vscode` and that's an amazing experience.

however, most of my workflow on `sublime-text`.

I wonder if there is any method to use marp on sublime-text? or, if any schedule to develop a sublime-text plug-in?

**Comment by yhatt** [comment, marked as answer]
Generally you can write Marp slide with any editor, by combining with [Marp CLI](https://github.com/marp-team/marp-cli) that is running at a terminal.

Marp team has no plan to make a package for Sublime Text because we are not familiar with Sublime Text and Python. Our ecosystem is open ecosystem so anyone can make a plugin for Marp if interested. (e.g. [Atom editor](https://atom.io/) (sunset) had [a Marp plugin made by community](https://atom.io/packages/atom-marp))

**Reply by dreistein543** [reply]
thank you for your reply. I have just tried Marp Cli, and it meets my need.
